packages/components/src/draggable/index.native.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+import Animated, { useSharedValue } from 'react-native-reanimated';
+
+/**
+ * Draggable component
+ *
+ * @param {Object}                                      props                       Component props.
+ * @param {JSX.Element}                                 props.children              Children to be rendered.
+ * @param {?number}                                     props.maxDistance           Maximum distance, that defines how far the finger is allowed to travel during a long press gesture.
+ * @param {?number}                                     props.minDuration           Minimum time, that a finger must remain pressed on the corresponding view.
```

Comment:
If these props are optional we should reflect them in the documentation, i.e.:
```suggestion
 * @param {?number}                                     [props.maxDistance]           Maximum distance, that defines how far the finger is allowed to travel during a long press gesture.
 * @param {?number}                                     [props.minDuration]           Minimum time, that a finger must remain pressed on the corresponding view.
```

**NOTE:** We could apply this to the callbacks as they're optional too.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/39551/commits/d8023e6d675ed58843275f865fb580fc402c4f9b#diff-4c96695e2f9beedb8860ca314946837c593df76d021a9167b7f90738ddcaabf0R12-R13

Reply:
>NOTE: We could apply this to the callbacks as they're optional too.

Good idea, I added them here https://github.com/WordPress/gutenberg/pull/39551/commits/d8023e6d675ed58843275f865fb580fc402c4f9b#diff-4c96695e2f9beedb8860ca314946837c593df76d021a9167b7f90738ddcaabf0R14-R16